### PR TITLE
export to word document

### PR DESCRIPTION
**Pull request by pacarole**

### CODE REVIEW

**Comment by skcain** [comment]
:squirrel:

File: jekyll/index.html [hunk 1]
```diff
@@ -60,5 +60,6 @@
 
   <section>
     <button data-control="gotoPage1Button" data-event-category="navigate" data-event-action="back">BACK</button>
+    <button data-control="saveAsWordDocButton" data-event-category="navigate" data-event-action="save as word doc">WORD DOCUMENT</button>
```
Comment (antislice) [inline jekyll/index.html:63]: is 'WORD DOCUMENT' going to be clear enough?

**Comment by antislice** [comment]
:ship:

File: jekyll/index.html (hunk 1 above)
Comment (pacarole) [inline jekyll/index.html:63]: I don't know if it's clear enough. I used the existing button style and that was the best thing that fit in the button. Ideally we think about how the feature works more holistically and make that change.

Comment (skcain) [inline jekyll/index.html:63]: I can also do some styling after you push live.

**Pull request merged**